change/@fluentui-react-1f0f8534-00d4-435e-ac52-e93820e744d7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "DetailsList: custom group header props such as styles are now correctly applied.",
```

Comment:
This change file should not be here, maybe just delete?

Reply:
Good catch, this file wasn't in my local environment so will just delete through here.